specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -259,6 +309,20 @@ model CertificateMetadata {
   name?: string;
 }
 
+@added(APIVersions.v2023_04_01_preview)
+@doc("Auto stop criteria for a test.")
+model AutoStopCriteria {
+  @doc("Auto stop is enabled or not")
+  autoStopEnabled?: boolean;
+
+  @doc("Value of auto stop criteria metric")
+  errorRate?: float32;
+
+  @doc("Time window in seconds for auto stop metric")
```

Comment:
auto-stop metric is it error rate. Should we have doc reflecting `specific error rate` instead of just saying it metric?

Also shall we use sentence casing ( add full stop at the end of the statement.)

Reply:
Thanks Sulabh, will discuss with Nikita regarding what should be the exact summary for these.